### ISSUE TITLE
Make the theme closer to the GTK one

### ISSUE THREAD

**Issue by Jogai** [the-zero885/dracula-Openbox#5]
There's quite a difference in the minimize/maximize/close buttons between openbox with this theme and gtk applications. You can see the difference when running chrome with the gtk theme or something like hourglass. Is this something you want to look into?

**Comment by the-zero885** [comment]
Yes, I just wanted to give it a different feel than the GTK theme. But yes, I can include a future variation that has the buttons that are a little closer to that of the GTK theme.
